### DISCUSSION TITLE
question about example "Simple_GP_Regression"(question)

### DISCUSSION THREAD

**Discussion by open2333** [cornellius-gp/gpytorch#1536] [Q&A] [answered]
Hi there, I am running the example "Simple_GP_Regression", I am using points without noise and I got this result
![image](https://user-images.githubusercontent.com/43056772/111720922-d16eb780-8899-11eb-91b9-5d623ebb5bc7.png)
May I ask how can I get the result that the confidence(variance) is zero at training points(observed data ) like the picture
![image](https://user-images.githubusercontent.com/43056772/111720939-ddf31000-8899-11eb-806f-3e6e025b0229.png)

**Comment by david-vicente** [comment, marked as answer]
I think that you can try to constrain the noise parameter.

``` python
likelihood = gpytorch.likelihoods.GaussianLikelihood(noise_constraint=gpytorch.constraints.Interval(0.0,0.0001))

```

**Reply by david-vicente** [reply]
when you force the noise parameter to zero the GP regression will work as an interpolator.

**Reply by gpleiss** [reply]
In general it's hard to get the noise parameter exactly to zero, as this would cause lots of numerical instabilities. However, if you make it something small (e.g. 1e-6) then you'll get a similar effect.
